### PR TITLE
Upload manifest fixes

### LINKED ISSUE #622: upload manifest modalities must be a list

### PR DESCRIPTION
**Pull request by alexpiet**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
- changes the data modalities in the upload manifest to be lists, in accordance with the aind-watchdog-service format
- Adds back `Window.session_name` and `Window.acqusition_datetime`, which are required fields for creating the upload manifest. They were deleted in https://github.com/AllenNeuralDynamics/dynamic-foraging-task/pull/597, and broke all upload manifest generation. 

### What issues or discussions does this update address?
- resolves #622 
- https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/517

### Describe the expected change in behavior from the perspective of the experimenter
- None

### Describe any manual update steps for task computers
- none

### Was this update tested in 446/447?
- [x] tested on my laptop
- [ ] tested in 446/447

### CODE REVIEW

**Comment by alexpiet** [comment]
@XX-Yin @hanhou This is ready for review

**Comment by XX-Yin** [comment]
It may not very relevant here. Will we use the similar upload manifest for uploading the old data?

**Comment by alexpiet** [comment]
> It may not very relevant here. Will we use the similar upload manifest for uploading the old data?

No. This upload manifest is just for new data.

File: src/foraging_gui/Foraging.py [hunk 1]
```diff
@@ -4237,9 +4239,9 @@ def _generate_upload_manifest(self,FIP=False):
...
                 'modalities':{
-                    'behavior':self.TrainingFolder.replace('\\','/'),
-                    'behavior-videos':self.VideoFolder.replace('\\','/'),
-                    'fib':self.PhotometryFolder.replace('\\','/')
+                    'behavior':[self.TrainingFolder.replace('\\','/')],
+                    'behavior-videos':[self.VideoFolder.replace('\\','/')],
+                    'fib':[self.PhotometryFolder.replace('\\','/')]
                     },
                 'schemas':[
                     os.path.join(self.MetadataFolder,'session.json').replace('\\','/'),
```
Comment (XX-Yin) [inline src/foraging_gui/Foraging.py:4247]: Can we add the data description if detected?

Comment (alexpiet) [reply src/foraging_gui/Foraging.py:4247]: I'm not sure if the watchdog service uses the data description. The data description will still get uploaded. Lets make that a separate issue so we can merge this in today

**Review by XX-Yin — APPROVED** [review]

**Pull request merged**